cmd/titus-inject-metadataproxy/main.go

```diff
@@ -50,14 +51,17 @@ func main() {
 	rootCmd.PersistentFlags().String("log-level", "info", "Log Level")
 
 	if err := v.BindPFlags(rootCmd.PersistentFlags()); err != nil {
-		panic(err)
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
This all looks good, looks like there's one more call to `BindPFlags` above this that could just `return err` too?